src/Microsoft.OData.Core/Microsoft.OData.Core.csproj

```diff
@@ -92,4 +92,15 @@
     <AdditionalFiles Include="PublicAPI/$(TargetFramework)/PublicAPI.Shipped.txt" />
     <AdditionalFiles Include="PublicAPI/$(TargetFramework)/PublicAPI.Unshipped.txt" />
   </ItemGroup>
+  <ItemGroup>
+    <Service Include="{508349b6-6b84-4df5-91f0-309beebad82d}" />

```

Comment:
I have the same question Clement had. You might had responded to him directly but I think it'd have helped to add the response here

Reply:
I've removed this. Must have been added when I ran the .tt files.